### DISCUSSION TITLE
Mullvad Browser not detected

### DISCUSSION THREAD

**Discussion by faxotherapy** [will-stone/browserosaurus#634] [Q&A]
Hi, any plan to detect Mullvad Browser? Currently, it's not detected. Is there a way to force detection, e.g. modifying a config file somewhere? Thanks for your help.

**Comment by will-stone** [comment]
Hi 👋 Please see the [supporting a browser or app](https://github.com/will-stone/browserosaurus/blob/main/guide/supporting-a-browser-or-app.md) guide. Thanks 🙂
